cfbs.json

```
@@ -1039,8 +1039,8 @@
       "tags": ["supported", "promise-type"],
       "repo": "https://github.com/cfengine/modules",
       "by": "https://github.com/tranchitella",
-      "version": "0.2.3",
-      "commit": "813fb3d172c8db5642ef69cd5e8ef32b264ef275",
+      "version": "0.2.4",
+      "commit": "3b9744c7173969a045f507d848acbf0fe5b78e6b",
```

It doesn't matter too much, but I like to use the merge commit to make it more obvious that this is the last commit in that repo / branch;

```suggestion
      "commit": "2ce7daa3c03e601ccc2686f16512a5d9739be9d7",
```